src/world.rs

```diff
@@ -47,15 +48,21 @@ pub struct World {
     archetypes: ArchetypeSet,
     /// Maps statically-typed bundle types to archetypes
     bundle_to_archetype: TypeIdMap<u32>,
+    id: usize,
 }
 
 impl World {
     /// Create an empty world
     pub fn new() -> Self {
+        static ID: AtomicUsize = AtomicUsize::new(0);
```

Comment:
Should we use `AtomicU64` here for insurance against long-running 32-bit systems?

Reply:
Changed it to `AtomicU64` so that 32-bit and 64-bit system have the same head room.